# Camera configuration for Recycling Robot

# Camera node parameters
camera_node:
  ros__parameters:
    stream_url: 'http://host.docker.internal:8554/feed.mjpg'
    fallback_device: 0
    width: 640
    height: 480
    fps: 10.0

# Classifier node parameters
classifier_node:
  ros__parameters:
    model_path: 'models/recycler.pth'
    confidence_threshold: 0.7
    inference_interval: 3.0

# Web node parameters
web_node:
  ros__parameters:
    port: 8000
    host: '0.0.0.0'
